### DISCUSSION TITLE
Finding Packet Drop for Ue to MEC communication

### DISCUSSION THREAD

**Discussion by mahbub-iut** [Unipisa/Simu5G#217] [Q&A]
I am simulating an application where NRUe sends BSM messages with a 10 HZ frequency to the MEC. I am simulating around 1000 UE in 10 minutes and  I can calculate the delay of these messages. However, is there any way that I can find out the number of packets dropped in the simulation?

**Comment by giovanninardini** [comment]
If you are interested in knowing the packet loss at the application level, then you should implement such a metric in your application.
One option is to record two statistics: 1) the number of message sent by the UE, and 2) the number of messages received by the MEC app. Then you can compute the difference as post processing operation.

**Comment by mahbub-iut** [comment]
Thank you for your response. I am talking about lower-level packet drop. I have a couple of questions related to that:

1. I found the statistics for uplink harqTxAttemptsUL and subsequent harqErrorRate_1st_UL,harqErrorRate_2nd_IL,harqErrorRate_3rd_UL, and harqErrorRate_4th_UL but finding it difficult to understand how it relates to Mac Layer packet drop or the number of retransmissions of packet?

2. Is there any way to find the Packet Reception rate (PRR) and Packet Error rate?
3. What is the minimum SINR value for the successful reception of the packet at the receiver?
4. I am also thinking about dynamically changing the configuration such as numerology, number of bands based on application layer delay, PRR, and  PER. Is it possible in Simu5G to change the gNodeB parameter while the simulation is running?
